frontend/packages/llama-stack-modular-ui/Makefile

```
@@ -0,0 +1,40 @@
+dev-install-dependencies: ## Install dependencies for frontend
+	cd frontend && npm install
+
+.PHONY: dev-bff
+dev-bff: ## Run only the bff in development mode
+	cd bff && make run MOCK_LS_CLIENT=true
+
+.PHONY: dev-frontend
+dev-frontend: ## Run only the frontend in development mode
+	cd frontend && URL_PREFIX='/llama-stack' PORT=9002 npm run start:dev
+
+.PHONY: dev-start
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)